doc/source/rayserve/overview.rst

```diff
@@ -0,0 +1,197 @@
+RayServe: Scalable and Programmable Serving
+===========================================
+
+.. image:: logo.svg
+    :align: center
+    :height: 250px
+    :width: 400px
+
+Overview
+--------
+
+RayServe is a scalable model-serving library built on Ray Core.
+
+RayServe allows you to:
+
+- **Framework Agnostic**:Use the same toolkit to serve everything from deep learning models 
+  built with frameworks like PyTorch or TensorFlow to scikit-learn models or arbitrary business logic.
+- **Simplify MLOps**: Configure your model serving with pure Python code - no more YAMLs or 
+  JSON configs.
+-  **A/B test models** (with zero downtime) by decoupling routing logic from response handling logic.
+- Leverage built-in **batching** and **SLO awareness** to help you meet your latency objectives.
+
+Since Ray is built on Ray, RayServe also allows you to **scale to many machines**
+and allows you to leverage all of the core Ray framework so you can deploy and scale on any cloud.
+
+.. note:: 
+  If you want to try out Serve, join our `community slack <https://forms.gle/9TSdDYUgxYs8SA9e8>`_ 
+  and discuss in #serve channel.
+
+Why RayServe?
+~~~~~~~~~~~~~
+
+There are generally two ways of serving machine learning applications at scale:
+
+1. The first is wrapping your application in a traditional web server. This approach
+is easy but hard to scale each component, and easily leading to high memory usage
+as well as concurrency issue.
+
+
+2. The other approach is to use a cloud-hosted solution like SageMaker or TFServing. 
+These solutions have high learning costs and lead to
+vendor lock-in.
+
+
+
+RayServe solves these problems by giving a user the ability to leverage the simplicity
+of deployment of a simple webserver but handles the complex routing, scaling, and testing logic
+necessary for production deployments.
```

Comment:
It's a little hard for me to skim this section (the "problems" are hidden). Maybe the first sentence should say "there are two ways ... but they both have fundamental limitations"

Reply:
Clarified.